src/precomputed-requestor.ts

```diff
@@ -4,8 +4,16 @@ import { IHttpClient } from './http-client';
 import { PrecomputedFlag, UNKNOWN_ENVIRONMENT_NAME } from './interfaces';
 import { Attributes } from './types';
 
+export interface PrecomputedResponseData {
+  salt: string;
+  subjectKey: string;
+  subjectAttributes: Attributes;
+}
```

Comment:
minor: this type still seems off to me. Why salt is here but actual response flags and bandits are not?

Reply:
cleaned up